chanbackup/single_test.go

```diff
@@ -124,8 +124,14 @@ func genRandomOpenChannelShell() (*channeldb.OpenChannel, error) {
 		isInitiator = true
 	}
 
+	var chanType channeldb.ChannelType
```

Comment:
should explicitly set chan type for tweak case

Reply:
I think you mean non-teak, but done.